src/main/java/core/basesyntax/model/FruitRecord.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.model;
+
+import java.util.NoSuchElementException;
+
+public class FruitRecord {
+    private final Operation operation;
+    private final String fruitName;
+    private final int amount;
+
+    public FruitRecord(Operation operation, String fruitName, int amount) {
+        this.operation = operation;
+        this.fruitName = fruitName;
+        this.amount = amount;
+    }
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public String getFruitName() {
+        return fruitName;
+    }
+
+    public int getAmount() {
+        return amount;
+    }
+
+    @Override
+    public String toString() {
+        return "FruitRecord{"
+                + "operation=" + operation
+                + ", fruitName='" + fruitName + '\''
+                + ", amount=" + amount
+                + '}';
+    }
+
+    public static Operation getOperationByFirstLetter(char symbol) {
```

Comment:
I'm not sure, but in this case it would be better to move this method to parse service impl, becouse they used to solve same problem

Reply:
moved getOperationByFirstLetter from FruitRecord class to ParseServiceImpl